### PR TITLE
Adding Accessibility Standards.

### PR DESCRIPTION
**Pull request by EdwinGuzman**

### CODE REVIEW

File: standards/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Accessibility Standards
+
+The Library provides resources that should be accessible to all patrons. When building front-end applications that patrons interact with, we MUST build the apps with accessibility in mind. This is a general overview and the Design Toolkit's [Developer Checklist](https://nypl.github.io/design-toolkit/resources/development-checklist.html) MUST be followed.
```
Comment (rhernand3z) [inline standards/accessibility.md]: I would promote saying we MUST build all applications in an accessibility first approach. Ensuring the development process integrates all a11y standards from the beginning. 🍔

File: standards/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Developers SHOULD be components that provide valid HTML over the framework's implementation. For example, when building a definition list `dl`, the only valid HTML allowed as children are `dt` and `dd`. React cannot return a component in the form of
+
+    <dt>Term</dt>
+    <dd>Definition</dd>
+
+and require the developer to wrap those elements in a `div` or `span`. Since this creates invalid HTML, it is preferable to not create a smaller component for that HTML and instead allow it to be part of a bigger component. It is better to write a big and accessible React component than smaller and inaccessible React components.
+
+### Tools
+
+Developers MUST run their apps through web accessibility tools.
```
Comment (rhernand3z) [inline standards/accessibility.md]: A reference of NYPL tools would be ideal. I think Ricardo tackled that in a Sprint way back in time. Maybe it is listed in the design toolkit?

**Comment by EdwinGuzman** [comment]
Thanks.

File: standards/accessibility.md (hunk 2 above)
Comment (EdwinGuzman) [inline standards/accessibility.md]: Updated. It is in the toolkit.

**Comment by kfriedman** [comment]
This is really great. Thanks for doing this!

File: standards/accessibility.md (hunk 1 above)
Comment (EdwinGuzman) [inline standards/accessibility.md]: Updated. Thanks.

**Comment by kfriedman** [comment]
Should this be merged? 😬

**Comment by EdwinGuzman** [comment]
MERGING.

**Pull request merged**